pkg/template/template.go

```diff
@@ -0,0 +1,173 @@
+package template
```

Comment:
also you should be able to reuse util for this logic

Reply:
removed.